### PR TITLE
Support python 3.8

### PR DESCRIPTION
**Pull request by omgoshjosh**
let's hope this doesn't break 3.6 or 3.7 support! 🤞

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-numpy==1.17.0
+numpy >=1.17.0,<2.0
 protobuf==3.6.1
-pandas==0.24.2
+pandas >=0.24.2,<2.0
```
Comment (Longi94) [inline requirements.txt:3]: Shouldn't this be `<1.0.0`?

File: carball/analysis/stats/utils/pandas_utils.py [hunk 2]
```diff
@@ -33,4 +33,4 @@ def sum_deltas_by_truthy_data(data_frame: pd.DataFrame, truthy_frames: pd.Series
     """
     truthy_frames = truthy_frames.rename('truthy')
     combined_data = pd.concat([truthy_frames, data_frame['game', 'delta'].rename('delta')], axis=1)
-    return combined_data.loc[combined_data['truthy'] == True].sum().rename(columns={'delta': 'true_values'}).delta
+    return combined_data.loc[combined_data['truthy'] == True].sum().loc['delta']
```
Comment (dtracers) [inline carball/analysis/stats/utils/pandas_utils.py:36]: Why this change?

Comment (omgoshjosh) [reply carball/analysis/stats/utils/pandas_utils.py:36]: to fix the 50+ tests that were broken. there we two issues with this line as i was debugging:
1. sum() returns a dataframe that didn't need to be renamed to get the delta
2. accessing the delta value by the `delta` index didn't seem to work the same way.

i am mostly unfamiliar with a lot of this stuff, so maybe there was a reason it was the way it was, but the API seems to have changed and i guess this is the "new" way to do it "right"?

File: requirements.txt (hunk 1 above)
Comment (omgoshjosh) [reply requirements.txt:3]: honestly, i don't know. i understand not wanting to frivolously update to the latest of upstream libraries for no reason, but 1.0 has been out for over 2 months and i am just trying to get python 3.8 support.

File: carball/analysis/stats/utils/pandas_utils.py (hunk 2 above)
Comment (dtracers) [reply carball/analysis/stats/utils/pandas_utils.py:36]: okay that makes sense.   But do you know what version it changed?

Because we are setting a new "minimum" pandas if this new way does breaks the old "minimum" pandas then we need to enforce the higher version

Comment (omgoshjosh) [reply carball/analysis/stats/utils/pandas_utils.py:36]: i don't think it does, otherwise we would have seen the tests break right? with my admittedly SUPER limited knowledge and exposure, my guess is they had a more flexible set of tools or maybe they had a less "intuitive" api where you had to rename an index to do an attribute access a la `df.thingToAccess`. maybe they also introduced certain restrictions around these things? here's the latest documentation on accessing an index by attribute name: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#attribute-access

Comment (dtracers) [reply carball/analysis/stats/utils/pandas_utils.py:36]: no that would not be the case because they all are installing pandas 1.03

Remember that the tests use a "fresh" install not an existing install.  So they do not run at the minimum.

Comment (dtracers) [reply carball/analysis/stats/utils/pandas_utils.py:36]: We should likely just change the minimum to the current version that is being run.  1.0.3
